sql-statements/sql-statement-rename-user.md

```diff
@@ -0,0 +1,94 @@
+---
+title: RENAME USER
+summary: TiDB 数据库中 RENAME USER 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-rename-user/','/docs-cn/dev/reference/sql/statements/rename-user/']
+---
+
+# RENAME USER
+
+`RENAME USER` 语句用于对已有用户进行重命名。
+
+## 语法图
+
+```ebnf+diagram
+RenameUserStmt ::=
+    'RENAME' 'USER' UserToUser ( ',' UserToUser )*
+
+UserToUser ::=
+    Username 'TO' Username
+
+Username ::=
+    StringName ('@' StringName | singleAtIdentifier)? | 'CURRENT_USER' OptionalBraces
+```
+
+## 示例
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE USER 'newuser' IDENTIFIED BY 'mypassword';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW GRANTS FOR 'newuser';
+```
+
+```
++-------------------------------------+
+| Grants for newuser@%                |
++-------------------------------------+
+| GRANT USAGE ON *.* TO 'newuser'@'%' |
++-------------------------------------+
+1 row in set (0.00 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+RENAME USER 'newuser' TO 'testuser';
+```
+
+```
+Query OK, 0 rows affected (0.08 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW GRANTS FOR 'testuser';
+```
+
+```
++--------------------------------------+
+| Grants for testuser@%                |
++--------------------------------------+
+| GRANT USAGE ON *.* TO 'testuser'@'%' |
++--------------------------------------+
+1 row in set (0.00 sec)
+```
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW GRANTS FOR 'newuser';
+```
+
+```
+ERROR 1141 (42000): There is no such grant defined for user 'newuser' on host '%'
+```
```

Comment:
````suggestion

```sql
CREATE USER 'newuser' IDENTIFIED BY 'mypassword';
```

```sql
Query OK, 1 row affected (0.02 sec)
```

```sql
SHOW GRANTS FOR 'newuser';
```

```sql
+-------------------------------------+
| Grants for newuser@%                |
+-------------------------------------+
| GRANT USAGE ON *.* TO 'newuser'@'%' |
+-------------------------------------+
1 row in set (0.00 sec)
```

```sql
RENAME USER 'newuser' TO 'testuser';
```

```sql
Query OK, 0 rows affected (0.08 sec)
```

```sql
SHOW GRANTS FOR 'testuser';
```

```sql
+--------------------------------------+
| Grants for testuser@%                |
+--------------------------------------+
| GRANT USAGE ON *.* TO 'testuser'@'%' |
+--------------------------------------+
1 row in set (0.00 sec)
```

```sql
SHOW GRANTS FOR 'newuser';
```

```sql
ERROR 1141 (42000): There is no such grant defined for user 'newuser' on host '%'
```
````